src/content/docs/style-guide/writing-docs/writer-workflow/github-intro.mdx

```diff
@@ -210,3 +212,155 @@ The life cycle of a viable issues candidate looks something like this:
+         0 = key does not expire
+      <n>  = key expires in n days
+      <n>w = key expires in n weeks
+      <n>m = key expires in n months
+      <n>y = key expires in n years
+Key is valid for? (0) 0
+Key does not expire at all
+Is this correct? (y/N) y
+
+GnuPG needs to construct a user ID to identify your key.
+
+Real name: Reli Relic
+Email address: rrelic@newrelic.com
+Comment: 
+You selected this USER-ID:
+    "Reli Relic <rrelic@newrelic.com>"
+```
+
+You should then be prompted to create a password (twice). Make sure to save this somewhere!
+You'll see some output which you can ignore. Now you can run this command to see your key:
+
+```
+gpg --list-secret-keys --keyid-format LONG
+```
+
+Which will return something like this:
+
+ ```
+reli ~ % gpg --list-secret-keys --keyid-format LONG
+gpg: checking the trustdb
+gpg: marginals needed: 3  completes needed: 1  trust model: pgp
+gpg: depth: 0  valid:   1  signed:   0  trust: 0-, 0q, 0n, 0m, 0f, 1u
+[keyboxd]
+---------
+sec   rsa4096/YOUR_KEY_ID 2023-07-18 [SC]
+      ABC123ABC123ABC123ABC123ABC123ABC123
+uid                 [ultimate] Reli Relic <rrelic@newrelic.com>
+ssb   rsa4096/ABC123ABC123 2023-07-18 [E]
+ ```
+
+ Now you can copy the value you see replaced by `YOUR_KEY_ID` in the above example. It might be a good idea to paste this somewhere temporarily since you'll be coming back to use it a few times.
+
+ Run this command and paste in your value in place of `YOUR_KEY_ID`
+
+ ```
+  gpg --armor --export YOUR_KEY_ID
+ ```
+
+ You should get an output that looks like this:
+
+ ```
+ -----BEGIN PGP PUBLIC KEY BLOCK-----
+
+numbersandletters378468734658734somanynumbers83475837485
+numbersandletters378468734658734somanynumbers83475837485
+numbersandletters378468734658734somanynumbers83475837485
+numbersandletters378468734658734somanynumbers83475837485
+numbersandletters378468734658734somanynumbers83475837485
+numbersandletters378468734658734somanynumbers83475837485
+numbersandletters378468734658734somanynumbers83475837485
+
+-----END PGP PUBLIC KEY BLOCK-----
+ ```
+
+ Copy the entire block including the begin and end comments. In github you'll navigate to your profile -> settings -> SSH and GPG keys. Here you can click "New GPG key", name it whatever you'd like, and paste in the previous block.
+ Hit save and you're done in github.
+
+ Now we'll adjust some settings in git by running these commands with `YOUR_KEY_ID` being replaced by your key from earlier:
+
+ ```
+git config --global user.signingkey YOUR_KEY_ID
+git config --global commit.gpgsign true
+ ```
+
+ Now you can run a test to see how things are working. You'll probably need to enter the password you created with gpg earlier. 
+
+ ```
+echo "test" | gpg --clearsign
+ ```
+
+ If you get an error and not a block of scrambled text (similar to the block you pasted in github) you can try running this command and then trying the test again:
+
+ ```
+ export GPG_TTY=$(tty)
+ ```
```

Comment:
i had to do this step. and when i open a new terminal window i have to do it each time in order for GPG to work. and when i do that i have to enter the passphrase to start using. 

running this command seemed to fix that (i was running zsh):

```bash
if [ -r ~/.zshrc ]; then echo -e '\nexport GPG_TTY=\$(tty)' >> ~/.zshrc; \
  else echo -e '\nexport GPG_TTY=\$(tty)' >> ~/.zprofile; fi
```